introduction/getting-started.qmd

```diff
@@ -124,12 +118,21 @@ Diagnosis is the process of simulating the design many times and calculating sum
 ::: {#lem-ch4num1} 
 
 Example design diagnosis
+Two-arm randomized experiment
+
+![Two-arm randomized experiment declaration](/figures/declaration-example){#fig-designannotated}
 
 ```{r}
 #| eval: false
 diagnose_design(declaration_4.1, sims = 100)
 ```
 
+
+
+:::
+
+The output of the diagnosis includes the diagnosand values (top row), such as bias of $-0.02$, and our uncertainty about the diagnosand value (bootstrapped standard error in parentheses in the bottom row). The uncertainty estimates tell us whether we have conducted enough simulations to precisely estimate the diagnosands. The fact that that the estimate of bias is $-0.02$ and the standard error is $0.03$ means that we cannot distinguish the amount of bias from no bias at all. 
+
```

Comment:
I don't think this is quite right. ::: should go after the table that has the diagnosis output. The diagnosis environment should contain the whole diagnosis, from text to the actual diagnosis output.

Reply:
Again, I changed this after looking at the final pdf file.